examples/auth/index.js

```diff
@@ -2,13 +2,13 @@
  * Module dependencies.
  */
 
-var express = require('../..');
-var bodyParser = require('body-parser');
-var hash = require('pbkdf2-password')()
-var path = require('path');
-var session = require('express-session');
+const express = require('../..'),
```

Comment:
We are moving toward using `standard` for formatting, which mandates multiple `const` statements.

Reply:
For reference: https://github.com/standard/standard

Reply:
I see. I will format them in the standard way.